src/app/core-components/welcome/welcome.component.ts

```diff
@@ -91,4 +104,44 @@ export class WelcomeComponent {
     this.onSiteVisitIdbService.setSelectedFromGUID(visit.guid);
     this.sharedDataService.createAssessmentModalOpen.next(true);
   }
+
+  openAddExampleModal() {
+    this.showAddExampleModal = true;
+  }
+
+  closeAddExampleModal() {
+    this.showAddExampleModal = false;
+  }
+
+  addExample() {
+    this.closeAddExampleModal();
+    this.loadingService.setLoadingMessage('Loading Example Data..');
+    this.loadingService.setLoadingStatus(true);
+    var request = new XMLHttpRequest();
+    request.open('GET', 'assets/example-data/ExampleData.json', true);
+    request.responseType = 'blob';
+    request.onload = () => {
+      var reader = new FileReader();
+      reader.readAsText(request.response);
+      reader.onloadend = async (e) => {
+        try {
+          let test = JSON.parse(JSON.stringify(reader.result));
+          let tmpBackupFile: BackupFile = JSON.parse(test);
```

Comment:
Is the JSON.parse() redundant? ` let tmpBackupFile: BackupFile = JSON.parse(JSON.stringify(reader.result));`?

Reply:
Resolved